ghost/core/core/server/models/index.js

```
@@ -1,87 +1,35 @@
+const glob = require('glob');
 
 // enable event listeners
 require('./base/listeners');
 
 /**
  * Expose all models
  */
-module.exports = {
-    // `base` file does not export a Base model
-    Base: require('./base'),
+exports = module.exports;
```

Biome (1.9.4)

[error] A global variable should not be reassigned.

Assigning to a global variable can override essential functionality.

(lint/suspicious/noGlobalAssign)